### PR TITLE
Print where raw cprofile lives

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #132866

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Sure

File: torch/_dynamo/convert_frame.py [hunk 1]
```diff
@@ -358,6 +358,7 @@ def profile_wrapper(*args: _P.args, **kwargs: _P.kwargs) -> _T:
             prof.dump_stats(profile_path)
         except PermissionError:
             log.warning("Cannot write to %s", str(profile_path))
```
Comment (Skylion007) [inline torch/_dynamo/convert_frame.py:360]: While you are editing here, might as well print exception info too?
```suggestion
            log.warning("Cannot write to %s", str(profile_path), exc_info=True)
```

**Pull request closed without merging**